playbook/lib/generators/kit/kit_generator.rb

```diff
@@ -5,12 +5,19 @@ class KitGenerator < Rails::Generators::NamedBase
   desc "This generator creates a new Playbook Kit"
   source_root File.expand_path("templates", __dir__)
   class_option :props, type: :array, default: []
+  class_option :rails, type: :boolean, default: false, desc: "Creates the boilerplate files for Rails"
+  class_option :react, type: :boolean, default: false, desc: "Creates the boilerplate files for React"
+  class_option :swift, type: :boolean, default: false, desc: "Creates the boilerplate files for Swift"
```

Comment:
The runway story had asked for a `skip` class_option but i think its better to implement each kit as a flag